README.md

```diff
@@ -1,249 +1,263 @@
-[Loom Video PT3](https://www.loom.com/share/39128df605f44263b1b4a5b8ddbcddb3)
-[Loom Video PT4](https://www.loom.com/share/7da5fc043d3149afb05876c28df9bd3d)
+<p><b>*NOTE:</b> "schedules" as used in endpoints refer to <b>childrens'</b> <b>enrollments</b> to <b>courses</b>; "schedule" specific endpoints join the <b>children</b>, <b>enrollments</b>, and <b>courses</b> tables and return data matching the given <b>profile_id</b></p>
 
 <h1>Profiles</h1>
 
 ```
 {
-  key: INCREMENT,
-  okta: STRING,
-  name: STRING,
-  email: STRING,
-  avatarUrl: STRING,
+  profile_id: INCREMENT (primary key, auto-increments, generated by database),
+  okta_id: STRING (unique),
+  name: STRING (required),
+  email: STRING (required),
+  role_id: INTEGER (required, foreign key),
+  avatarUrl: STRING (required, defaults to: 'https://i.stack.imgur.com/frlIf.png'),
 }
 ```
 
-<h1>Admins</h1>
+| Method   | URL                         | Description                                                                                                              |
+| -------- | --------------------------- | ------------------------------------------------------------------------------------------------------------------------ |
+| [GET]    | /profile/                   | Returns an array of all existing profiles.                                                                              |
+| [GET]    | /profile/:okta_id/       | Returns the profile object with the specified `okta_id`.                                                                       |
+| [GET]    | /profiles/users/:profile_id <b>(BUG: /profiles/users route does not exist; app.js only connects to /profiles/user)</b> | Returns an array filled with event objects that contains information based on profile_id and role_id.                    |
+| [GET]    | /profile/role/:role_id <b>(BUG: does not return any data)</b>      | Returns an array filled with event objects that contain information based on role_id for all profiles of a role_id type. |
+| [POST]   | /profile/                   | Requires a name, password, and email. Registers a new user.                                                              |
+| [PUT]    | /profile/                   | Returns an event object with the specified `okta`. Updates specific profile.                                             |
+| [DELETE] | /profile/:okta_id/             | Returns an event object with the specified `okta`. Deletes specific profile.                                             |
+#### User:
+<p>These endpoints are user-focused. As opposed to the more flexible Profile endpoints where <b>profile_id</b> must be specified, these endpoints retrieve data specific only to the user profile that is making the API request by using the logged-in user's <b>profile_id</b>.</p>
 
-```
-{
-  id: INCREMENT,
-  user_id: INTEGER,
-}
-```
+| Method | URL              | Description                                                             |
```

Comment:
I like how you set this up for users to read. As a visual learner, this helps anyone understand the CRUD easier and what it returns.